src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -20,12 +24,18 @@
 
 @RestController
 @RequestMapping("/api/books")
+@Api
 public class BookController {
 
     @Autowired
     private BookRepository bookRepository;
 
     @GetMapping("/{author}")
+    @ApiOperation(value = "Given an author returns the book", response = Book.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, response = Book.class, message = ""),
+            @ApiResponse(code = 404, message = "Book not found")
+    })
     public Book findByAuthor(@PathVariable String author) {
```

Comment:
You should document the endpoint parameters too :) 

Also, all endpoints should be documented